kubernetes-server-mock/src/main/java/io/fabric8/kubernetes/client/server/mock/KubernetesAttributesExtractor.java

```diff
@@ -150,7 +126,9 @@ private static AttributeSet extract(Matcher m) {
       if (!Strings.isNullOrEmpty(kind)) {
 
         //Poor mans to singular.
-        if (kind.endsWith("s")) {
+        if (kind.endsWith("ses")) {
```

Comment:
Maybe its time to use a real utility for converting to singular like: https://github.com/sundrio/sundrio/blob/95c2b11f7b842bdaa04f61e8e338aea60fb38f70/codegen/src/main/java/io/sundr/codegen/functions/Singularize.java

Reply:
But for now this will do.